src/Kiota.Builder/Refiners/RubyRefiner.cs

```diff
@@ -33,16 +37,16 @@ public override void Refine(CodeNamespace generatedCode)
             new ("SerializationWriterFactory", "microsoft_kiota_abstractions"),
         };
         private static readonly Tuple<string, string>[] defaultNamespaces = new Tuple<string, string>[] { 
-            new ("SerializationWriter", "microsoft_kiota_abstractions"),
+            new ("SerializationWriter", "microsoft_kiota_serialization"),
```

Comment:
those 3 classes belong to abstractions, effectively they should be in abstractions/serialization